app/views/users/show.html.erb

```diff
@@ -27,6 +27,7 @@
     <% end %>
   </ul>
   <div class="card-body d-flex flex-row" id="navigate">
+  <%= link_to "Create new post", data: { turbo: false }, new_user_post_path(@user) %>
     <%= link_to 'View more posts', user_posts_path(@user), class: 'btn btn-lg' %>
     <%= link_to 'Back', users_path(@user), class: 'btn btn-lg btn-danger' %>
   </div>
```

Comment:
- [ ] Good job fixing the issues raised by the previous reviewer, however you have an error when the user clicks on each user in the app. Due to this I am unable to see if all requested changes were implemented. Kindly fix this.
The error is due to a syntax issue in your link_to call in the div with the id "navigate." Specifically, it seems there's a misplaced parenthesis

Video 
[cinnamon-2023-12-16T234323+0200.webm](https://github.com/Mesi21/visionOfLife/assets/81932478/c6756981-2c9d-492c-b3d4-6517694b8d26)